src/pylife/materialdata/woehler/fatigue_data.py

```diff
@@ -160,6 +164,18 @@ def set_fatigue_limit(self, fatigue_limit):
 
         return self
 
+    def irrelevant_runouts_dropped(self):
+        '''Checks if data has several pure runout load levels and drops all except max pure runout level
```

Comment:
Consider changing to "Make a copy of the instance with irrelevant pure runout levels dropped."

Reply:
Yes, it sounds much better! THX